cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup conol oil
```

Comment:
Looks like you mis-spelled conola oil in your commit. Please fix the spelling and re-commit.